packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/diagrams/ProjectWithDiagramUnsynchroniszedUploadControllerTests.java

```diff
@@ -191,6 +193,14 @@ private void checkImportedProject(String response) {
         assertThat(representation.get().getLayoutData().edgeLayoutData().size()).isEqualTo(1);
         assertThat(representation.get().getLayoutData().edgeLayoutData().get(representation.get().getEdges().get(0).getId())).isNotNull();
 
+        assertThat(representation.get().getNodes().get(1).getCustomizedStyleProperties()).hasSize(1);
```

Comment:
how many times should `representation.get()` be repeated? On top of that, we have both a custom way to navigate in diagrams and tons of custom assertions for diagrams, they should be used. I don't understand a single check in the code here.

Reply:
I've updated the test. Be careful with the quality of the code within our tests next time.